packages/nhsuk-frontend/src/nhsuk/components/file-upload/file-upload.mjs

```diff
@@ -0,0 +1,492 @@
+import { closestAttributeValue } from '../../common/closest-attribute-value.mjs'
+import { ConfigurableComponent } from '../../configurable-component.mjs'
+import { ElementError } from '../../errors/index.mjs'
+import { I18n } from '../../i18n.mjs'
+
+/**
+ * File upload component
+ *
+ * @preserve
```

Comment:
We can drop this. There's a custom [GOV.UK Frontend Babel config](https://github.com/alphagov/govuk-frontend/pull/3958) to preserve or remove unnecessary comments

```suggestion
```